inbox/events/microsoft/parse.py

```diff
@@ -692,8 +703,19 @@ def parse_event(
     ]:
         assert master_event_uid
         assert event["type"] in ["exception", "synthesizedCancellation"]
-
-    uid = event["id"]
+    log = get_logger()
+    log.info(
+        "Parsing event",
+        event_ical_id=event["iCalUId"],
+        event_uid=event["id"],
+        created_at=event["createdDateTime"],
+        title=event["subject"],
+    )
+    ical_uid = event["iCalUId"]
+    event_id = event["id"]
+    created_date_time = datetime.datetime.fromisoformat(
+        event["createdDateTime"]
+    )
```

Comment:
You might want to follow up with a PR to remove this later once we are sure that the implementation works well because this will log a lot and historically we were trying to keep logging at bay in this project.